src/Phinx/Db/Adapter/PdoAdapter.php

```diff
@@ -167,6 +167,36 @@ public function insert(Table $table, $row)
         $stmt = $this->getConnection()->prepare($sql);
         $stmt->execute(array_values($row));
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function bulkinsert(Table $table, $rows)
+    {
+        $sql = sprintf(
+            "INSERT INTO %s ",
+            $this->quoteTableName($table->getName())
+        );
+
+        $current = current($rows);
+        $keys = array_keys($current);
+        $sql .= "(". implode(', ', array_map(array($this, 'quoteColumnName'), $keys)) . ") VALUES";
+
+        $objTmp = (object) array('aFlat' => array());
+        array_walk_recursive($rows, create_function('&$v, $k, &$t', '$t->flat[] = $v;'), $objTmp);
```

Comment:
Thank you for your review !

I tried to simply coded in one line.
But there was not enough consideration to Phinx.

Fix it !

Thx.